plot_widget.py

```diff
@@ -64,6 +64,21 @@ def plot_dcm(self, path):
         logger.info("plot_dcm completed within PlotWidget")
         return self.axes.axis() != (0.0, 1.0, 0.0, 1.0)
 
+    def force_plot_dcm(self, path):
```

Comment:
reuse plot_dcm with the added attribute force. This can be set to a default of false

Reply:
I think this is easier to test, and not the same as the normal way to open. We can add some other stuff later and logging should also be distinguished from the normal one as this is force open.

Reply:
Talking to jack he reckons it should be the same function as well and pass force in. logging can be personalized by formatting the force parameter in